DFC.ServiceTaxonomy.VersionComparison/Models/Parts/ObjectValue.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+
+namespace DFC.ServiceTaxonomy.VersionComparison.Models.Parts
+{
+    public class ObjectValue
+    {
+        public string? Html { get; set; }
+        public string? Text { get; set; }
+        public string? Value { get; set; }
+
+        public override string ToString()
+        {
+            if (!string.IsNullOrWhiteSpace(Html))
+            {
+                return Html;
+            }
+            if (!string.IsNullOrWhiteSpace(Text))
+            {
+                if (Text.Equals("true", StringComparison.InvariantCultureIgnoreCase) ||
```

Comment:
Could possibly use bool.TryParse, so we don't have to compare strings twice?

Reply:
Amended